testdata/PerpDeployHaltTrading_Success.yaml

```diff
@@ -0,0 +1,52 @@
+---
+version: 2
+interactions:
+    - id: 0
+      request:
+        proto: HTTP/1.1
+        proto_major: 1
+        proto_minor: 1
+        content_length: 271
+        host: api.hyperliquid-testnet.xyz
+        body: '{"action":{"haltTrading":{"coin":"test:BTC","isHalted":true},"type":"perpDeploy"}}'
```

Comment:
how could you get a success here with a mock coin `test:BTC`? Hope you didn't manually edit this yaml 🙃